spec/tags/gbm.md

```diff
@@ -0,0 +1,21 @@
+The Google Business Message channel may be used after its activation on Zenvia platform.
+
+To activate Google Business Message you need to be registered as a partner with Google Business Message and get an account information configured on Zenvia platform.
+
+**Get in touch with Zenvia consultants to start your account creation.**
+
+Webhooks allow you to receive events in the configured URL. [Learn more here.](#tag/Webhooks)
+
+## Google Business Message sender and recipient
+
+When you send some message for one contact using Google Business Message channel:
+
+* Recipient: is the phone number of contact
```

Comment:
O id não é o telefone não! É agente igual no RCS, não?

Reply:
![3](https://user-images.githubusercontent.com/44481544/117206596-6f2a3180-adc9-11eb-8a76-629e6fd83130.png)

Reply:
só um detalhe, fica melhor "the contact id". e na parte de baixo vc pos sender duas vezes

Reply:
Boa

Reply:
![1](https://user-images.githubusercontent.com/44481544/117306519-21a7d600-ae56-11eb-8891-66c03da6a498.png)